README.md

```diff
@@ -42,31 +43,33 @@ The deployment scripts are organized in the following structure:
 The scripts tend to follow a similar structure, and are either simple helpers or wrappers for the `forge script` command.
 
 General options will include:
+
 - `-n`: The network to run on, e.g. `baklava` or `alfajores`
 - `-u`: The upgrade number, e.g. `MU01`
 
 Check the script file for more details on usage but here's a quick overview:
 
 ```bash
 # Clean the broadcast folder, will remove all broadcast files pertaining to that network and upgrade combination
-> yarn clean -n baklava -u MU01 
+> yarn clean -n baklava -u MU01
 
 # Show the list of deployed contracts and their addresses
 > yarn show -n baklava -n MU01
+{"name":"BreakerBoxProxy","address":"0xB881aF21C5A9ff8e8d5E4C900F67F066C6CB7936"}
```

Comment:
I mean that's just an example of using the script, and realistically if you run MU01 it will show that because the proxy was deployed as part of MU01 even tho we don't need it anymore. And yeah I didn't add this explicitly it just came from reverting the commit that removed everything.